packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -217,33 +217,29 @@ const DatePickerField = (props) => {
 
   let ariaDescriptionIds;
 
-  if (help && error && isInvalid) {
+  if (error && isInvalid) {
     ariaDescriptionIds = `${datePickerId}-error ${datePickerId}-help`;
   } else {
-    if (help) {
-      ariaDescriptionIds = `${datePickerId}-help`;
-    }
-
-    if (error && isInvalid) {
-      ariaDescriptionIds = `${datePickerId}-error`;
-    }
+    ariaDescriptionIds = `${datePickerId}-help`;
   }
 
+  const inputAriaDescriptionId = inputAttributes && inputAttributes['aria-describedby'];
```

Comment:
should we assign only inputAttributes['aria-describedby'] here ?

Reply:
Yes, as we only want to concatenate and use `aria-describedby` for the current scenario.

Reply:
yes, but i was asking like this  'const inputAriaDescriptionId = inputAttributes['aria-describedby'];'

Reply:
this would throw up if inputAttributes in undefined.